modules/configuring-a-proxy-after-installation-ocm.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-proxy-after-installation-ocm_{context}"]
+= Configuring a proxy after installation using {cluster-manager}
+
+You can use {cluster-manager-first} to add a cluster-wide proxy configuration to an existing {product-title} cluster in a Virtual Private Cloud (VPC). You can enable a proxy only for clusters that are deployed by using the Customer Cloud Subscription (CCS) model.
+
+You can also use {cluster-manager} to update an existing cluster-wide proxy configuration. For example, you might need to update the network address for the proxy or replace the additional trust bundle if any of the certificate authorities for the proxy expire.
+
+[IMPORTANT]
+====
+The cluster applies the proxy configuration to the control plane and compute nodes. While applying the configuration, each cluster node is temporarily placed in an unschedulable state and drained of its workloads. Each node is restarted as part of the process.
+====
+
+.Prerequisites
+
+* You have an {product-title} cluster that uses the Customer Cloud Subscription (CCS) model.
+* Your cluster is deployed in a VPC.
+
+.Procedure
+
+. Navigate to {cluster-manager-url} and select your cluster.
+
+. Under the *Virtual Private Cloud (VPC)* section on the *Networking* page, click *Edit cluster-wide proxy*.
+
+. On the *Edit cluster-wide proxy* page, provide your proxy configuration details:
+.. Complete at least one of the following fields:
```

Comment:
Thanks for highlighting this. I struggled with this one for a while before I submitted the PR. None of the following seem adequate:

- I considered "Add a value to at least one of the following fields". But, then I wondered if it should be "value" or "values" depending on how many fields you update.

- I thought about using "Update at least one of the following fields" to get around the plural challenge. But, if there is nothing in the field to begin with, then it is not being updated, but added.

- "Enter at least one of the following fields" seems a bit incomplete, because you are explicitly entering something into the field.

I will have a think about how to address this.

Reply:
I have decided on "Enter a value for at least one of the following fields:".